src/backend/expungeservice/__init__.py

```diff
@@ -3,3 +3,4 @@
 from . import database
 from . import endpoints
 from . import expunger
+from . import request
```

Comment:
It's generally good style to leave a newline at the end of the file. Not important, but thought I'd mention. You can setup your editor to automatically add a newline and remove trailing whitespace. You can also setup git to add newlines at the end of file and remove trailing whitespace. 

<img width="301" alt="Screenshot 2019-06-08 19 22 32" src="https://user-images.githubusercontent.com/19519317/59154256-fd9da900-8a22-11e9-9173-06f997e54b94.png">

Reply:
Gotcha. I'll fix that and look into the automatic solution for the future, thanks!